### PR TITLE
Fixed #118 and #185, added #106, check for deno.land/x urls in readme, basic auditing

### LINKED ISSUE #118: Remove No README warning when there is a REAMDME.md
**Is your feature request related to a problem? If so, please describe.**
When i ran eggs publish, it said `No README found at project root`, although I do have a README.md, this should probably be covered

**Describe the solution you'd like**
README.md autodetection supported as well as just README

**Describe alternatives you've considered**
N/A

**Additional context**
GitHub's default readme file is also README.md

### PR DESCRIPTION
**Pull request by MartonDev**
Changes:
- [x] Fixed issue #118. The publish command now actually checks if a readme file is present instead of testing in the files array
- [x] Fixed issue #185 and #67. The website now shows the correct entry file
- [x] Added feature #106. (copy import/entry url)
- [x] Added checking for deno.land/x urls in readme. If an "old import url" (an url with deno.land/x/MOD_NAME) is found, it will warn the user that they can change it to x.nest.land/MOD_NAME...
- [x] Basic auditing. Added auth panel, malicious package display. TODO (not in this PR): more auditing, permissions, warnings

### CODE REVIEW

**Comment by MartonDev** [comment]
@tbaumer22 ready for review

File: web/src/components/package/PackageDetail.vue [hunk 1]
```diff
@@ -177,6 +177,19 @@
...
+                <p class="panel-heading">
+                  <font-awesome-icon class="icon-margin-right" :icon="['fas', 'shield-alt']" />Audit
+                </p>
+                <div class="panel-block" v-if="malicious">
+                  <font-awesome-icon class="icon-margin-right" :icon="['fa', 'biohazard']" />
+                  Malicious module: use at your own risk
+                </div>
+                <div class="panel-block" v-else>
+                  <font-awesome-icon class="icon-margin-right" :icon="['fa', 'lock']" />
+                  The module is safe
```
Comment (tbaumer22) [inline web/src/components/package/PackageDetail.vue:190]: I don't think we should display that it is "safe," as it requires someone reporting that the module is malicious for us to flag it as such. Instead, I think we should just remove that section all together and only warn the user if it is malicious.

File: web/src/components/package/PackageDetail.vue [hunk 2]
```diff
@@ -177,6 +177,19 @@
                   Published on: {{ packageInfo.createdAt | formatDate }}
                 </div>
               </nav>
+              <nav class="panel">
+                <p class="panel-heading">
+                  <font-awesome-icon class="icon-margin-right" :icon="['fas', 'shield-alt']" />Audit
+                </p>
+                <div class="panel-block" v-if="malicious">
+                  <font-awesome-icon class="icon-margin-right" :icon="['fa', 'biohazard']" />
+                  Malicious module: use at your own risk
```
Comment (tbaumer22) [inline web/src/components/package/PackageDetail.vue:186]:
```suggestion
                  Flagged as malicious
```

File: web/src/components/package/PackageDetail.vue (hunk 1 above)
Comment (MartonDev) [reply web/src/components/package/PackageDetail.vue:190]: We can do that for now, however I want to display permissions, etc. there too in the future

**Comment by MartonDev** [comment]
Resolving what you mentioned, and adding some new small changes too

File: web/src/components/package/PackageDetail.vue (hunk 1 above)
Comment (tbaumer22) [reply web/src/components/package/PackageDetail.vue:190]: I see. That would be neat. We'll have to figure out a good way to organize that when the time comes.

**Pull request merged**